libs/adapter/payment-service/bitpay-payment-service.ts

```diff
@@ -0,0 +1,39 @@
+import { Environment, PaymentService } from '@pl-oss/core';
+import {
+  Client, Currency, Env, Models, Tokens,
+} from 'bitpay-sdk';
+
+function createBitpayClient({ bitpayMerchantId, bitpayPrivateKey, runtime }: Environment) {
```

Comment:
Can't assume that Environment will have those properties. 

Let's use individual arguments instead of object destructuring